Sources/AzureAppConfiguration/AzureAppConfiguration.swift

```diff
@@ -0,0 +1,91 @@
+import CryptoKit
```

Comment:
Not sure how necessary this is, but I think we generally have some sort of generic Bottle Rocket comment block at the top of each file.